homeassistant/components/zwave_js/__init__.py

```diff
@@ -132,12 +134,21 @@
 DATA_DRIVER_EVENTS = "driver_events"
 DATA_START_CLIENT_TASK = "start_client_task"
 
-CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Optional(CONF_INSTALLER_MODE, default=False): cv.boolean,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
 
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the Z-Wave JS component."""
-    hass.data[DOMAIN] = {}
+    hass.data[DOMAIN] = config.get(DOMAIN, {})
```

Comment:
It's not worth it now imo with only this hidden expert option. If we add other config options in the future it would make sense but it would be very easy to refactor then

Reply:
Side note: Whenever I start a comment with "side note:" it means my comment isn't completely rated to the code change or it's something that can be addressed in the future. 😉